pos_anonymous_customer/models/res_partner.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    available_in_pos = fields.Boolean(
+        string='Available for POS',
```

Comment:
How do you see default value : 
1> currently : default : none
2> could be default = is_customer (to initialize correctly existing partners).
3> or maybe =True, if customer is created via the UI : not sure if it is easy to catch.

In anyways, an api@depend, setting available_in_pos to False if is_customer is set to False could be great ? Don't you think ?